test/Microsoft.AspNetCore.Mvc.Razor.Test/RazorViewEngineOptionsTest.cs

```diff
@@ -12,6 +13,65 @@ namespace Microsoft.AspNetCore.Mvc.Razor
     public class RazorViewEngineOptionsTest
     {
         [Fact]
+        public void AreaViewLocationFormats_ContainsExpectedLocations()
+        {
+            // Arrange
+            var services = new ServiceCollection().AddOptions();
+            var areaViewLocations = new[]
+            {
+                "/Areas/{2}/MvcViews/{1}/{0}.cshtml",
+                "/Areas/{2}/MvcViews/Shared/{0}.cshtml",
+                "/MvcViews/Shared/{0}.cshtml"
+            }.AsEnumerable();
```

Comment:
As discussed offline, Roslyn must be getting confused. Though IntelliSense shows the `Assert.Equal()` overload is the `IEnumerable<T>` one, the test actually makes calls to `Assert.Equal<object>(array, list)` or something similar. Easy fixes include `Assert.Equal<string>(array, list)` or `Assert.Equal(array, list, StringComparer.Ordinal)`.